xml/obs_ag_spider_identification.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC
+  "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.docbook.org/xml/4.5/docbookx.dtd"
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<section id="_obs_spider_identification">
+ <title>Spider Identification</title>
+ <para>
+   OBS is hiding specific parts/pages of the application from search crawlers (duckduckgo, google etc.), mostly for performance reasons.
+   Which user-agent strings are identified as crawlers is configured by the <ulink url="https://github.com/monperrus/crawler-user-agents/blob/master/crawler-user-agents.json">upstream crawler-user-agents.json</ulink> file.
```

Comment:
Why is an upstream file? It's not created by default in our code base? It should be I guess...